app/src/main/java/com/example/recipeapp/activities/MainActivity.java

```diff
@@ -23,7 +23,7 @@
 public class MainActivity extends AppCompatActivity {
 
     private static final String TAG = "MainActivity";
-    public NavController navController;
+    public static NavController navController;
```

Comment:
I'll follow up on this but sometimes we want to be careful about using `static` in android. Not sure at the moment though.